### ISSUE TITLE
iOS17 privacy policy

### ISSUE THREAD

**Issue by KKakadiyaKashyap** [hackiftekhar/IQKeyboardManager#2025]
Apple required App Privacy manifests until 2025. So GoogleSignIn-iOS SDK that contains an app privacy manifest must be signed.
I would like to know when you adopt this change.

**Comment by rick711003** [comment]
I also would like to know any update

**Comment by hackiftekhar** [comment]
App Privacy manifests file is already there. If you have any additional things then please ask.

**Comment by app4g** [comment]
@hackiftekhar thank you for the update. I was updating my app when I got Apple email about the need for PrivacyInfo.xcprivacy
I then complied and add it to the project.

However, the project was unable to proceed past Compilation/Archive due to the error as described here https://github.com/hackiftekhar/IQKeyboardManager/issues/2004

I then tried various means to get it working, including removing the input scripts in the [CP] Copy Resources Build Phase eg:

Original:
```
INPUT:
${PODS_ROOT}/Target Support Files/Pods-BLE/Pods-BLE-resources.sh
${PODS_ROOT}/IQKeyboardManagerSwift/IQKeyboardManagerSwift/PrivacyInfo.xcprivacy

OUTPUT:
${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/PrivacyInfo.xcprivacy
```

Updated:
```
INPUT:
${PODS_ROOT}/Target Support Files/Pods-BLE/Pods-BLE-resources.sh
${PODS_ROOT}/IQKeyboardManagerSwift/IQKeyboardManagerSwift/PrivacyInfo.xcprivacy

OUTPUT:
${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/PrivacyInfo.xcprivacy
```

This then enabled the compilation. But when archived and sent to Apple, seems like the PrivacyInfo.xcprivacy from IQKeyboardManagerSwift overwrote my app's own PrivacyInfo file.

I confirmed this by
1) Removing IQKeyboardManagerSwift from my PodFile, doing a pod install and then removing import statements in my app
2) Archive the app,
3) Go into the App, (show package contents) and then looked at the PrivacyInfo.xcprivacy file. 
4) This file does contain my edits and is the file for my App.

My question is, how can I get the behaviour of the new merge (resource_bundle) into a version 6.x? Is there any plans to do that or will I need to roll my own fork?

TQ so much

**Comment by app4g** [comment]
Just tried installing via SPM and it allowed me to install version 7.x
Archiving it and then looking into the Archive, we now have IQKeyboardManagerSwift with it's own bundle with it's own PrivacyInfo.xcprivacy in it

<img width="459" alt="Screenshot 2024-03-15 at 4 23 17 PM" src="https://github.com/hackiftekhar/IQKeyboardManager/assets/107654902/56fe1d06-6204-4e20-8071-e97cad80d8c8">

**Comment by hackiftekhar** [comment]
This issue is already addressed in this PR https://github.com/hackiftekhar/IQKeyboardManager/pull/2005 and already fixed with 7.0.1 version.

**Comment by WGMB** [comment]
Hi, please create new release version with the latest changes and a Cocoapods release as well.
